### DISCUSSION TITLE
Sample Worflows do not work

### DISCUSSION THREAD

**Discussion by TechyGuy99** [elsa-workflows/elsa-core#5056] [Q&A]
Ran the Docker compose
went to localhost:13000 and logged in.

Imported backend-api sample.
![image](https://github.com/elsa-workflows/elsa-core/assets/150186692/917c91a5-49f6-4912-b340-522d43b4a0f2)

executed curl as per documentation:
curl --location 'https://localhost:5001/workflows/users/2'

I get the following error, Same thing with the hello-world.json and every sample i have tried.
I have also tried running the one under guides sample folder.   Nothing works.

>curl --location 'https://localhost:5001/workflows/users/2'
curl: (3) URL rejected: Port number was not a decimal number between 0 and 65535

**Comment by glime-ai** [comment]
Based on the information provided, it seems like you are trying to execute a workflow using a curl command that references `https://localhost:5001`, but the Docker container is running on port `13000`. The error message you're seeing (`curl: (3) URL rejected: Port number was not a decimal number between 0 and 65535`) indicates that there's an issue with the port number in the URL you're using.

To resolve this issue, you should use the correct port number that corresponds to the running Docker container. If the Docker container for Elsa Workflow is running on port `13000`, you should adjust the curl command to use `http://localhost:13000` instead of `https://localhost:5001`.

Here's the corrected curl command:

```
curl --location 'http://localhost:13000/workflows/users/2'
```

Please note that you should use `http` instead of `https` unless you have set up SSL for the Docker container, which is not indicated in the provided context.

If you continue to experience issues, please ensure that the workflow is published and that the URL you are using matches the HTTP trigger URL of the workflow you imported. You can find the correct URL in the Elsa Dashboard under the workflow's HTTP Endpoint settings.

**Comment by CD1010** [comment]
The server is on 5001 because that is what i started it on.  I did publish.  None of samples working.

**Comment by sfmskywalker** [comment]
@TechyGuy99 If your Elsa Server is hosted on port 13000, then you also need to send your HTTP requests to that port, like the bot mentioned.

@CD1010 Can you show me the URL to the sample that you tried?

**Comment by glime-ai** [comment]
Based on the context provided, it seems like there is a mismatch between the port number used in the cURL command and the port number on which the Elsa Workflow instance is actually running. If you are running the Elsa Docker container and accessing it via `localhost:13000`, then you should also send your HTTP requests to port `13000` instead of `5001`.

Please try updating your cURL command to use the correct port number that matches the one configured for your Docker container. Here is the updated cURL command:

```shell
curl --location 'http://localhost:13000/workflows/users/2'
```

Make sure to use `http` instead of `https` unless you have specifically configured SSL for your Docker container, and replace `5001` with `13000` which is the port you mentioned you are using to access the Elsa Dashboard.

If you continue to experience issues, please ensure that the workflow has been published and that the HTTP Endpoint is correctly configured to respond to the path `/workflows/users/2`.
